setup.py

```diff
@@ -4,10 +4,18 @@
 # Full license can be found in License.md
 # -----------------------------------------------------------------------------
 
+"""Setup routine for sami2py.
+
+Note
```

Comment:
```suggestion
Notes
```

Standard section name is Notes.